build-configs.yaml

```diff
@@ -139,6 +139,12 @@ trees:
 
 fragments:
 
+  crypto:
+    path: "kernel/configs/crypto.config"
+    configs:
+      - 'CONFIG_CRYPTO_TEST=m'
```

Comment:
Is there any advantage in having this built as a module rather than built-in?

Reply:
It is a symbol which can only be set as module.